internal/provider/data_source_gitlab_users.go

```diff
@@ -212,6 +212,12 @@ var _ = registerDataSource("gitlab_users", func() *schema.Resource {
 							Type:        schema.TypeString,
 							Computed:    true,
 						},
+						"namespace_id": {
+							Description: "The ID of the users namespace. Requires admin token to access this field. Available since GitLab 14.10.",
```

Comment:
If my other comment stands, then same thing here.

Reply:
It mentions the admin limitation already or what are you referring to?

Reply:
Sorry, I meant the `users -> user's` suggestion.